src/upgrade-compatibility-tool/install.md

```diff
@@ -0,0 +1,123 @@
+
+## Prerequisites
+
+See [prerequisites]({{site.baseurl}}/upgrade-compatibility-tool/prerequisites.html) for more information.
+
+{:.bs-callout-info}
+You can run the Upgrade Compatibility Tool in any operating system. There is no requirement to run the Upgrade Compatibility Tool where your Magento instance is located. It is necessary for the Upgrade Compatibility Tool to have access to the source code of the Magento instance. For example, you can install the tool on one server and point it at your Magento installation on another server.
+
+If you are running the Upgrade Compatibility Tool against a Magento instance with large modules and files, the tool might require a high amount of RAM, at least 2GB RAM.
+
+### Recommended actions
+
+Magento best practice is not to have 2 modules with the same name, if this happens the Upgrade Compatibility Tool will show a segmentation fault error in which case you have to analyze each module independently with the option `-m`:
+
+```bash
+bin/uct upgrade:check /(instance_path) --coming-version=2.4.1 -m /(module_path)
+```
+
+If you get memory issues while executing the Upgrade Compatibility Tool it is recommended to use the `-m` command to run the tool against a specific module.
+
+## Download the Upgrade Compatibility Tool
+
+To download the Upgrade Compatibility Tool, run the following command:
+
+```bash
+composer create-project magento/upgrade-compatibility-tool uct  --repository https://repo.magento.com
+```
+
+As the Upgrade Compatibility Tool is an independent tool, if you try to run:
+
+```bash
+composer require magento/upgrade-compatibility-tool
+```
+
+It might add the Upgrade Compatibility Tool as a dependency for a Magento project.
+
+## Install
+
+To install the Upgrade Compatibility Tool, you must install the necessary prerequisites:
+
+*  Magento access keys
+*  Composer
+*  Node.js (only required to check GraphQL compatibility)
+
+Refer to the [Upgrade Compatibility Tool install]({{site.baseurl}}/upgrade-compatibility-tool/install.html#install) page.
+
+### Magento access keys
+
+You must have [Magento access keys]({{site.baseurl}}/marketplace/sellers/profile-information.html#access-keys) to download and use the Upgrade Compatibility Tool. Add your Magento access keys to your `auth.json` file, which is located at `~/.composer` by default.
+
+{:.bs-callout-warning}
+Check your **COMPOSER_HOME** environment variable to see where the `auth.json` file is located.
+
+The **public key** corresponds to the _username_ whereas the **private key** is the _password_:
+
+### Example of Magento access keys
+
+```json
+    "http-basic": {
+        "repo.magento.com": {
+            "username": "YOUR_MAGENTO_PUBLIC_KEY",
+            "password": "YOUR_MAGENTO_PRIVATE_KEY"
+        }
+    },
+```
+
+### Composer
+
+Clone the [upgrade-compatibility-tool](https://github.com/magento-commerce/safe-upgrade-tool) repository and run `composer install` in your terminal to install dependencies.
```

Comment:
This should be upgrade-compatibility-tool

Reply:
@sgonzalezmangana repo name has not changed yet

Reply:
I'll keep it as is for the moment, if repo name changes, we should update this part of the docs